Panic when deploying after kick merge

#111 results in a panic in render during deployment.
```
panic: ':roomid' in new path '/rooms/:roomid/kick/:userid' conflicts with existing wildcard ':id' in existing prefix '/rooms/:id'
```
[Apparently](https://github.com/gin-gonic/gin/issues/1301), we just need to rename `:roomid` to `:id` in route.

> Basically, both routes need to have the wildcard in the same position, and the wildcards need to be named the same thing.

--- a/routes/router.go
+++ b/routes/router.go
@@ -67,7 +67,7 @@ func SetupRouter(router *gin.Engine) *gin.Engine {
 				roomRouter.PATCH("/:id", invalidateCacheURI, UpdateRoom)
 				roomRouter.PATCH("/:id/connect", invalidateCacheURI, ConnectUserToRoom)
 				roomRouter.PATCH("/:id/disconnect", invalidateCacheURI, DisconnectUserFromRoom)
-				roomRouter.PATCH("/:roomid/kick/:userid", invalidateCacheURI, KickUserFromRoom)
+				roomRouter.PATCH("/:id/kick/:userid", invalidateCacheURI, KickUserFromRoom)
 			} else {
 				roomRouter.GET("/:id", GetRoom)
 				roomRouter.PATCH("/:id", UpdateRoom)

